lib/addons/flagger/my_new_addon.ts

```diff
@@ -0,0 +1,65 @@
+// filename=lib/kubevious_addon.ts
+import 'source-map-support/register';
+import * as blueprints from '../../../lib';
+import { Construct } from 'constructs';
+import { setPath } from '../../utils';
+
+/**
+ * User provided options for the Helm Chart
+ */
+export interface FlaggerAddOnProps extends blueprints.HelmAddOnUserProps {
+  version?: string,
```

Comment:
why version is here? is it not in the base interface already?

Reply:
Example from outline I did not remove yet.